packages/fast-viewer-react/src/components/viewer/index.tsx

```diff
@@ -255,4 +250,10 @@ export default class Viewer extends React.Component<IViewerProps, IViewerState>
             return this.renderIFrame();
         }
     }
+
+    private addLoadEventListenerToItems(items: HTMLCollection | NodeListOf<HTMLLinkElement>): void {
+        for (let i: number = 0, itemsLength: number = items.length; i < itemsLength; i++) {
```

Comment:
let here can be const

Reply:
you could also alternatively use for...of since you aren't using the index for anything

Reply:
'i' gets redefined as it gets incremented, so let is accurate here.

Reply:
won't fix - can't use for of